src/NzbDrone.Api/REST/RestModule.cs

```diff
@@ -214,7 +214,10 @@ protected TResource ReadResourceFromRequest(bool skipValidate = false)
         private PagingResource<TResource> ReadPagingResourceFromRequest()
         {
             int pageSize;
-            int.TryParse(Request.Query.PageSize.ToString(), out pageSize);
+            if (!int.TryParse(Request.Query.PageSize.ToString(), out pageSize))
+            {
+                pageSize = 100000;
```

Comment:
im just handling the case, where nothing is passed.... so i.e. there is no pageSize param... I think...

so when the parseInt fails.. not if the pageSize sent is actually 0... if its actually 0.. then it will use 10 not 100000....